packages/react-card/src/components/CardHeader/renderCardHeader.tsx

```diff
@@ -11,11 +11,13 @@ export const renderCardHeader_unstable = (state: CardHeaderState) => {
   return (
     <slots.root {...slotProps.root}>
       <slots.image {...slotProps.image} />
-      <slots.content {...slotProps.content}>
-        <slots.header {...slotProps.header} />
-        <slots.description {...slotProps.description} />
-      </slots.content>
-      <slots.action {...slotProps.action} />
+      {slots.content && (
+        <slots.content {...slotProps.content}>
+          <slots.header {...slotProps.header} />
```

Comment:
Not related to this PR, but there are required slots rendered inside an optional slot. Is that expected?
cc @khmakoto

Reply:
CCing @andrefcdias

Reply:
Yes, but now that you mention it, behavior is weird. Content only exists due to our policy of "every container is a slot", even though it's only there for styling the content as is designed in Fluent.
I'll add this problem to my list of stuff to resolve.